api/task_test.go

```diff
@@ -203,3 +236,64 @@ func TestTask_decodeJSON(t *testing.T) {
 		})
 	}
 }
+
+func TestTask_RunOption(t *testing.T) {
+	cases := []struct {
+		name        string
+		path        string
+		option      string
+		expectError bool
+	}{
+		{
+			"happy path run now",
+			"/v1/tasks?run=now",
```

Comment:
Are the run options available for `/v1/tasks`? or `/v1/tasks/:task`?

Reply:
Thanks for catching this! It should be `/v1/tasks/:task`. Let me update!